### DISCUSSION TITLE
Check out my Age Playground

### DISCUSSION THREAD

**Discussion by mingfang** [apache/age#1633] [Show and tell]
Hi All,
Please check it out Age Playground.
https://age-playground.rebelsoft.com/

The goal is to release three components.
1-Javascript Age driver that let's you use pure cypher, without the sql wrapper like in the Playground
2-REST server/middleware to enable remote Cypher execution. Again like in the Playground
3-An embeddable version of the Playground so that Age data and/or visualization can be easily embedded within your site

Look forward to your feedback.

**Comment by jrgemignani** [comment]
@mingfang Thank you! I will pass this discussion thread to the other members of our team.

**Comment by markgomer** [comment]
Hey @mingfang, this is awesome. I'm looking forward to it. Please do share any news
